fuse_models/src/acceleration_2d.cpp

```diff
@@ -51,7 +51,8 @@ namespace fuse_models
 Acceleration2D::Acceleration2D() :
   fuse_core::AsyncSensorModel(1),
   device_id_(fuse_core::uuid::NIL),
-  tf_listener_(tf_buffer_)
+  tf_listener_(tf_buffer_),
+  throttled_callback_(std::move(std::bind(&Acceleration2D::process, this, std::placeholders::_1)))
```

Comment:
Just curious, is the `std::move` needed here? I thought since `std::bind` is returning a temporary anyway, it was intrinsically moveable.

Reply:
Removed `std::move`